tests/python/frontend/tflite/test_forward.py

```diff
@@ -624,17 +661,23 @@ def _test_greater(data):
     """ One iteration of greater """
     return _test_elemwise(math_ops.greater, data)
 
-def _test_forward_elemwise(testop):
+def _test_forward_elemwise(testop, quantized=False):
```

Comment:
Do you think it will be cleaner if we have separate function _test_forward_elemwise_quantized

Reply:
I agree we separate int8 / fp32 two isolated functions, because we have others functions, like fully_connected, we could do it the same too.

Reply:
Ok, makes sense. I will change it.